src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -443,5 +489,59 @@ private static byte GetSeverityNumber(LogLevel logLevel)
                 return 1;
         }
     }
+
+    private static int SanitizeCategoryName(byte[] buffer, int cursor, ref int cursorStartIdx, ref int validNameLength, string categoryName)
+    {
+        if (categoryName.Length > (1 << 8) - 1)
```

Comment:
nit
```suggestion
        if (categoryName.Length > 255)
```

Reply:
This is the number here:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/faaa8b56eae0196754ec1162d4cc7a8253d45baa/src/OpenTelemetry.Exporter.Geneva/MessagePackSerializer.cs#L60

I'll update the code here to use const to store this number for better readability.
(Though left shift and direct assignment will result in exactly the same assembly, I guess the reason why we use left shift here was to better manifest MessagePack serialization protocol.)
https://github.com/msgpack/msgpack/blob/master/spec.md#str-format-family
str 8 stores a byte array whose length is upto (2^8)-1 bytes:
+-----+----------+===+
| 0xd9 |YYYYYYYY| data |
+-----+----------+===+